torrentleech_monitor/monitor.py

```diff
@@ -90,6 +93,9 @@ def _get_torrents(show_name, season_number, episode_number, session):
                 results_list = [t.find('a')['href'] for t in table.find_all('td', 'quickdownload')]
                 sizes_list = [t.string for t in table.find_all('td') if t.string and
                               ('GB' in t.string or 'MB' in t.string)]
+                seeders_list = [t.get_text() for t in table.find_all('td', 'seeders')]
```

Comment:
Use `int` conversion here (instead of `map` in `sort_by_seeders`). 
No use in keeping this information as str..